diff --git a/src/main/osd/osd.c b/src/main/osd/osd.c
index da39035e9..ee15f7584 100644
--- a/src/main/osd/osd.c
+++ b/src/main/osd/osd.c
@@ -882,7 +882,7 @@ static timeDelta_t osdShowArmed(void)
     } else {
         ret = (REFRESH_1S / 2);
     }
-    displayWrite(osdDisplayPort, 12, 7, DISPLAYPORT_ATTR_NONE, "ARMED");
+    displayWrite(osdDisplayPort, 10, 7, DISPLAYPORT_ATTR_NONE, "AKRAMODE");
 
     return ret;
 }
diff --git a/Makefile b/Makefile
index 4f54cbd18..724036524 100644
--- a/Makefile
+++ b/Makefile
@@ -302,7 +302,7 @@ CPPCHECK        = cppcheck $(CSOURCES) --enable=all --platform=unix64 \
                   -I/usr/include -I/usr/include/linux
 
 
-TARGET_BASENAME = $(BIN_DIR)/$(FORKNAME)_$(FC_VER)_$(TARGET)_$(REVISION)
+TARGET_BASENAME = $(BIN_DIR)/$(FORKNAME)_$(FC_VER)_$(TARGET)-$(shell date +'%Y%m%d')
 
 #
 # Things we will build
 